### PR TITLE
Update of the terminology

### PR DESCRIPTION
**Pull request by beurdouche**
Aligning the terminology across the two documents after discussion.

### CODE REVIEW

**Comment by beurdouche** [comment]
Ok, I think this is ready, please have a look.. I believe the changes are reasonably concise, the use of client seems better in some places than participant was and doesn't feel odd to me.

File: draft-ietf-mls-protocol.md [hunk 1]
```diff
@@ -260,48 +262,48 @@ This protocol is designed to execute in the context of a Messaging Service (MS)
...
-The protocol algorithms we specify here follow. Each algorithm specifies both (i) how a participant
-performs the operation and (ii) how other participants update their state based on it.
+The protocol algorithms we specify here follow. Each algorithm specifies both (i) how a client
+performs the operation and (ii) how other clients update their state based on it.
 
 There are four major operations in the lifecycle of a group:
 
 * Adding a member, initiated by a current member;
-* Adding a member, initiated by the new member;
+* Adding a member, initiated by the future member;
```
Comment (beurdouche) [inline draft-ietf-mls-protocol.md]: I'll remove that line in a separate commit, since we currently don't have UserAdd

File: draft-ietf-mls-protocol.md [hunk 2]
```diff
@@ -118,27 +118,27 @@ Instructions are on that page as well. Editorial changes can be
 managed in GitHub, but any substantive change should be discussed on
 the MLS mailing list.
 
-A group of agents who want to send each other encrypted messages needs
+Users who want to send each other encrypted messages needs
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md]: Subject/verb agreement "Users ... needs".  Re-introducing "A group of..." would fix this.

File: draft-ietf-mls-protocol.md [hunk 3]
```diff
@@ -118,27 +118,27 @@ Instructions are on that page as well. Editorial changes can be
...
-security, but are nonetheless efficient enough for heavy use over
-low-bandwidth networks.
+Channels implementing the Double Ratchet enjoy fine-grained forward secrecy
+as well as post-compromise security, but are nonetheless efficient
+enough for heavy use over low-bandwidth networks.
 
 For a group of size greater than two, a common strategy is to
 unilaterally broadcast symmetric "sender" keys over existing shared
-symmetric channels, and then for each agent to send messages to the
+symmetric channels, and then for each Member to send messages to the
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md]: Let's not capitalize terms, just be consistent.

File: draft-ietf-mls-protocol.md [hunk 4]
```diff
@@ -154,6 +154,8 @@ RFC EDITOR PLEASE DELETE THIS SECTION.
 
 draft-04
 
+- Updating the language to be similar to the Architecture document. (\*)
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md]: This doesn't need a `(\*)`, since it's not a breaking change.

File: draft-ietf-mls-protocol.md (hunk 3 above)
Comment (bifurcation) [reply draft-ietf-mls-protocol.md]: Likewise in a bunch of places below.

File: draft-ietf-mls-protocol.md [hunk 5]
```diff
@@ -260,48 +262,48 @@ This protocol is designed to execute in the context of a Messaging Service (MS)
...
-  participant can download initialization keys for other participants.
+* A directory to which clients can publish initialization keys and download
+  initialization keys for other participants.
 
 
 # Protocol Overview
 
-The goal of this protocol is to allow a group of participants to exchange confidential and
-authenticated messages. It does so by deriving a sequence of secrets and keys known only to group members. Those
+The goal of this protocol is to allow a group of clients, members of a group, to exchange confidential and
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md]: This reads awkwardly.  Suggest deleting ", members of a group,"

File: draft-ietf-mls-protocol.md [hunk 6]
```diff
@@ -1183,13 +1185,13 @@ A group member generates this message by requesting a UserInitKey
 from the directory for the user to be added, and encoding it into an
 Add message.
 
-The new participant processes Welcome and Add messages together as
-follows:
+The client about to join the group processes Welcome and Add
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md]: I might rephrase this as "The client joining the group..."

File: draft-ietf-mls-protocol.md [hunk 7]
```diff
@@ -1207,15 +1209,15 @@ the signature on the message,  then updates its state as follows:
 The update secret resulting from this change is an all-zero octet
 string of length Hash.length.
 
-On receipt of an Add message, new participants SHOULD send an update
-immediately to their key. This will help to limit the tree structure
+After processing an Add message, new members SHOULD send an Update
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md]: This would read better as singular, "... the new member SHOULD..."

File: draft-ietf-mls-protocol.md (hunk 3 above)
Comment (beurdouche) [reply draft-ietf-mls-protocol.md]: I just removed capitalization everywhere.

File: draft-ietf-mls-protocol.md (hunk 2 above)
Comment (beurdouche) [reply draft-ietf-mls-protocol.md]: Fixed.

File: draft-ietf-mls-protocol.md (hunk 4 above)
Comment (beurdouche) [reply draft-ietf-mls-protocol.md]: Fixed.

File: draft-ietf-mls-protocol.md (hunk 5 above)
Comment (beurdouche) [reply draft-ietf-mls-protocol.md]: Fixed.

File: draft-ietf-mls-protocol.md (hunk 6 above)
Comment (beurdouche) [reply draft-ietf-mls-protocol.md]: Fixed.

File: draft-ietf-mls-protocol.md (hunk 7 above)
Comment (beurdouche) [reply draft-ietf-mls-protocol.md]: Fixed

File: draft-ietf-mls-protocol.md (hunk 1 above)
Comment (beurdouche) [reply draft-ietf-mls-protocol.md]: I did that as well.
